### ISSUE TITLE
error while using 12cmd [NotImplementedError: Cannot convert a symbolic Tensor]

### ISSUE THREAD

**Issue by salmaShahid** [douglas125/SpeechCmdRecognition#16]
![image](https://user-images.githubusercontent.com/51890862/213873445-e20a91f2-1652-4105-8da5-d2189062e4b1.png)

While running speech model attention based model; I got this error;
kindly help me to solve this error

**Comment by douglas125** [comment]
Are you using the versions required in `requirements.txt` ?

I think the tensorflow version might have to be 2.3.0

**Comment by salmaShahid** [comment]
Yes i am using the same version; tensorflow==2.3.4 (as write in requirement.txt)
But it still generating this error


Here is my colab notebook

https://colab.research.google.com/drive/1ckppY0hvvnlUi3a74NVMsYy5fIrIhbGM?usp=sharing


Kindly guide me how to solve this error

**Comment by douglas125** [comment]
I updated everything to tensorflow 10.
The notebook runs from start to end for me now.
Please let me know if it solves the problem - no more obsolete packages.
